eZ/Publish/Core/MVC/Symfony/Security/EventListener/SecurityListener.php

```diff
@@ -213,6 +214,15 @@ public function onKernelRequest( GetResponseEvent $event )
             return;
         }
 
+        try
+        {
+            $this->repository->getUserService()->loadUser( $this->repository->getCurrentUser()->contentInfo->id );
+        }
+        catch ( NotFoundException $e )
+        {
+            $this->repository->setCurrentUser( $this->repository->getUserService()->loadUser( 10 ) );
```

Comment:
Yes, this is one of the bits that are NOT ok. Will certainly improve this, if by any chance I get the green light for the general idea of the fix.